### PR TITLE
Fix typo on tsl -> tls for v1.0 of protocol

### PR DESCRIPTION
**Pull request by ferd**
Also bump app version

### CODE REVIEW

**Comment by stillinbeta** [comment]
⛵ shipit

File: src/ranch_proxy_encoder.erl [hunk 1]
```diff
@@ -5,7 +5,7 @@
 -include("ranch_proxy.hrl").
 
 -type opts() :: [{negotiated_protocol, binary()}
-                |{protocol, sslv3 | tslv1 | 'tlsv1.1' | 'tlsv1.2'}
+                |{protocol, sslv3 | tlsv1 | 'tlsv1.1' | 'tlsv1.2'}
```
Comment (stillinbeta) [inline src/ranch_proxy_encoder.erl]: I think what's most impressive is that this mistake was made *twice* CtrlP strikes again!

**Comment by dpiddy** [comment]
Any way to test?

**Comment by ferd** [comment]
`curl --tlsv1.0 -I -v $URL` will yield an empty response as the proxy fails to run anything.

**Comment by dpiddy** [comment]
I mean, is there a test that could be added to the project to help prevent this from regressing?

**Comment by ferd** [comment]
Probably, we'd need to make sure these kinds of connections work and can be set up I believe. We should add them eventually, but I needed this merged ASAP to fix existing bugs for now.

File: src/ranch_proxy_encoder.erl (hunk 1 above)
Comment (ypaq) [inline src/ranch_proxy_encoder.erl]: :tada:

**Comment by ypaq** [comment]
Is dialyzer check exhaustive enough to check it?

**Comment by ferd** [comment]
Well possibly, if we had not also typoed the type spec

**Pull request merged**